plugin.video.tvvlaanderen/addon.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.tvvlaanderen" name="TV Vlaanderen" version="1.0.0" provider-name="Michaël Arnauts">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+        <import addon="script.module.dateutil" version="2.6.0"/>
+        <import addon="script.module.inputstreamhelper" version="0.5.1"/>
+        <import addon="script.module.pysocks" version="1.6.8" optional="true"/>
+        <import addon="script.module.requests" version="2.22.0"/>
+        <import addon="script.module.routing" version="0.2.0"/>
+        <import addon="script.module.pyjwt" version="1.6.4"/>
+        <import addon="inputstream.adaptive" version="2.4.3"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon_entry.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="nl_NL">Bekijk Live en on-demand TV via je TV Vlaanderen abonnement.</summary>
```

Comment:
Is this nl only? If so please set the language tag so it can be hidden for foreign users

Reply:
Hmm, there are also French and English channels, but it's a Belgian service (Belgium has 3 official languages). Is there a way to set a tag based on country?

Reply:
Nop, only language. It's okay to keep it as is. Thanks.